_source/_posts/2022-09-26-java-jetty-oauth.md

```diff
@@ -0,0 +1,1017 @@
+---
+layout: blog_post
+title: "Get Started with Jetty, Java, and OAuth"
+author: andrew-hughes
+by: contractor
+communities: [java]
+description: "Learn how to use Java and embedded Jetty to create a simple servlet-based web service. Then see how to do it with Spring Boot."
+tags: [java, jetty, oauth2]
+tweets:
+- "Jetty is a highly scalable Java web server and servlet engine. This tutorial shows how to use it with servlets and @springboot."
+- "Want to learn how to use Java and embedded @JettyProject to develop REST APIs? This tutorial is for you!"
+- "Build a Java REST API with @JettyProject and learn how to lock it down with OAuth 2.0 in this tutorial."
+image: blog/featured/okta-java-short-bottle-headphones.jpg
+type: conversion
```

Comment:
```suggestion
type: conversion
github: https://github.com/oktadev/okta-spring-boot-jetty-example
```

Reply:
This makes a "code" icon show at the top of the post. It's useful for when I do demos or screencasts.